packages/edit-widgets/src/components/sidebar/widget-areas.js

```diff
@@ -47,7 +47,7 @@ export default function WidgetAreas( { selectedWidgetAreaId } ) {
 			<div className="edit-widgets-widget-areas__top-container">
 				<BlockIcon icon={ blockDefault } />
 				<div>
-					<p>{ description }</p>
+					<p dangerouslySetInnerHTML={ { __html: description } } />
```

Comment:
Let's add a comment of why we need to use `dangerouslySetInnerHTML` here explicitly and why it's okay in this context.

Reply:
Sure. Added a comment. Feel free to change it if it doesn't make sence.

Reply:
Thanks! But I mean why we need to dangerously set `innerHTML` instead of just rendering `description` as children.

Reply:
It could be worth using the `safeHTML` function from the `dom` package on the `description` if it's going to be rendered using `dangerouslySetInnerHTML`.

While I think it should generally be a safe value, I don't think the block editor's UI has full control over where the value comes from. It wouldn't be outside the world of possibility for a plugin to allow a user to set this value.

Reply:
Changed the comment to describe why `dangerouslySetInnerHTML` is used and added `safeHTML`.